modules/UI/recording/Recording.js

```diff
@@ -196,27 +196,37 @@ function _showStopRecordingPrompt(recordingType) {
 
 /**
  * Moves the element given by {selector} to the top right corner of the screen.
+ * Set additional classes that can be used to style the selector relative to the
+ * state of the filmstrip.
+ *
  * @param selector the selector for the element to move
  * @param move {true} to move the element, {false} to move it back to its intial
  * position
  */
 function moveToCorner(selector, move) {
-    const {
-        remoteVideosCount,
-        remoteVideosVisible,
-        visible
-    } = APP.store.getState()['features/filmstrip'];
-    selector.toggleClass(
-        'with-filmstrip',
-        Boolean(remoteVideosCount && remoteVideosVisible && visible));
-
     let moveToCornerClass = "moveToCorner";
     let containsClass = selector.hasClass(moveToCornerClass);
 
     if (move && !containsClass)
         selector.addClass(moveToCornerClass);
     else if (!move && containsClass)
         selector.removeClass(moveToCornerClass);
+
+    const {
+        remoteVideosCount,
+        remoteVideosVisible,
+        visible
+    } = APP.store.getState()['features/filmstrip'];
+    const filmstripWasHidden = selector.hasClass('without-filmstrip');
+    const filmstipIsOpening = filmstripWasHidden && visible;
+    selector.toggleClass('opening', filmstipIsOpening);
+
+    selector.toggleClass('with-filmstrip', visible);
+    selector.toggleClass('without-filmstrip', !visible);
+
+    const oneOnOne = Boolean(remoteVideosVisible && remoteVideosCount);
```

Comment:
It is for the case where the entire filmstrip is displayed but there are no videos, so the labels need to go to the right edge. I'll do a renaming pass to help clear this up.